src/store-tree/index.ls

```diff
@@ -40,45 +37,34 @@ class StoreTree extends StoreNode
 
 
   $is-loading: ->
-    for own key of @_children when @[key].$is-loading!
-      return yes
-    no
+    @_children |> values |> any (.$is-loading!)
```

Comment:
That's a lot more elegant. If you insist on type correctness, I would go that route. What I am saying is that you can just omit this and nobody will ever notice any difference, except your code is slightly faster, smaller, and simpler. When in doubt, leave it out. Use types where they actually help you. Don't add useless boilerplate just to satisfy a non-existing type checker  :)